src/java.base/share/classes/sun/security/ssl/BaseSSLSocketImpl.java

```diff
@@ -41,6 +41,11 @@
  * Methods are defined final to ensure that they are not accidentally
  * overridden in SSLSocketImpl.
  *
+ * There used to be a finalize() implementation, but decided that was
```

Comment:
Since you haven't pushed yet, maybe:

    There used to be a finalize() implementation that sent close_notify's, but decided that was not needed.  An application should close its sockets and not let them get garbage collected without closing them.  

    The underlying native resources are handled by the Socket Cleaner.

Reply:
Thanks for the rewording.  Updated.

Reply:
> Thanks for the rewording. Updated.

I made one more tweak that reads better.

Reply:
Yes, it looks better.  Updated.  Thanks!

Reply:
Looks good, thanks.